### DISCUSSION TITLE
Is autogluon 1.2 official yet?

### DISCUSSION THREAD

**Discussion by michelidiego** [autogluon/autogluon#4760] [General]
Hi! I know you released 1.2 version and I can install it and everything, but when I check the "What's new" section in the documentation site , I don't see any mention of it. (https://auto.gluon.ai/stable/whats_new/index.html)

So I just wanted to be sure that I'm using the latest official version of autogluon.

Thanks!

**Comment by shchur** [comment]
Hi, thanks for pointing this out, the documentation needs to be updated. Turning this into an issue for more visibility. #4769

v1.2 is the latest official release.

**Reply by michelidiego** [reply]
thank you for your reply
